heron/executor/src/python/heron_executor.py

```diff
@@ -415,9 +452,19 @@ def _wait_process_std_out_err(self, name, process):
       do_print("%s stderr: %s" %(name, process_stderr))
 
   def _run_process(self, name, cmd, env_to_exec=None):
-    do_print("Running %s process as %s" % (name, ' '.join(cmd)))
-    return subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
-                            env=env_to_exec)
+    if self.pyheron_instance_binary in cmd[0]:
+      # enable profiling for pyheron_instance
+      if env_to_exec is None:
+        env = {}
+      else:
+        env = env_to_exec.copy()
+      env["PEX_PROFILE"] = "true"
```

Comment:
This profiling is available only if we use local scheduler, correct? If that is the case do we need it?